docs/data/tree-view/simple-tree-view/customization/customization.md

```diff
@@ -11,11 +11,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
 
 <p class="description">Customize the rendering of your Tree View.</p>
 
-## Custom icons, border and animation
+## Basics of customization
+
+### Custom icons
+
+Use the `defaultCollapseIcon`, `defaultExpandIcon`, and `defaultEndIcon` props to customize the Tree View icons. You can use either icon libraries, such as Material Icons, or create your own.
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
+
+### Custom toggle animations
+
+Use the `TransitionComponent` prop on the `TreeItem` to pass a component that handles your animation.
+
+{{"demo": "CustomAnimation.js", "defaultCodeOpen": false}}
+
+### Styling
+
+You can use the `treeItemClasses` to target the internal elements of the TreeView.
+
+{{"demo": "CustomStyling.js"}}
+
+### Setting custom content
+
+You can use the `ContentComponent` prop to replace the `TreeItem` with a custom component
+
+{{"demo": "CustomContentTreeView.js"}}
+
+## Common examples
+
+### Connection border
+
+Target the `treeItemClasses.group` class to add connection borders between the Tree View items.
+
+{{"demo": "BorderedTreeView.js", "defaultCodeOpen": false}}
+
+### File explorer
+
+You can customize the Tree View to accomodate more complex use cases:
 
 {{"demo": "CustomizedTreeView.js"}}
 
-## ContentComponent prop
+### Gmail clone
```

Comment:
```suggestion
### Gmail clone

Google's Gmail side nav is potentially one of the most famous tree view components.
The demo below shows how to replicate it.
```

Feel like it's a bit weird not having at least a small description text there :)